demisto_sdk/commands/format/format_module.py

```diff
@@ -87,7 +87,8 @@ def format_manager(input: str = None,
         int 0 in case of success 1 otherwise
     """
     if input:
-        files = get_files_in_dir(input, ['json', 'yml', 'py', 'md'])
+        if isinstance(input, str):
+            files = get_files_in_dir(input, ['json', 'yml', 'py', 'md'])
```

Comment:
condition is redundant as we cast the input to string or passing None (which will not get here)
```suggestion
        files = get_files_in_dir(input, ['json', 'yml', 'py', 'md'])
```